src/components/SubscribeBanner/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import background from '../../../static/img/summit-landing/subscribe/subscribe-banner-bg.png'
+import './styles.scss';
+
+const SubscribeBanner = () => (
+
+    <section className='subscribe-banner-wrapper' style={{ backgroundImage: `url(${background}` }}>
+        <div className="container subscribe-banner-container">            
+            <span className='subscribe-banner-title'>
+            Subscribe to our newsletter <br/>
+            & keep up to date with the latest <br/>
+            News about the Summits.
+            </span>
+            <a href="#" className="subscribe-banner-button">
```

Comment:
this need to be a prop
check https://tipit.avaza.com/project/view/188886#!tab=task-pane&groupby=ProjectSection&view=vertical&task=3432630&stafilter=NotComplete&fileview=grid